test/test_integration.py

```diff
@@ -496,3 +496,27 @@ def update_graph(n_clicks):
         button.click()
         time.sleep(2)
         self.snapshot('candlestick - 2 click')
+
+    def test_latex(self):
+        app = dash.Dash(__name__)
+        app.layout = html.Div([
+
+            html.Label('Graph'),
+            dcc.Graph(
+                id='graph',
+                figure={
+                    'data': [{
+                        'x': [1, 2, 3, 4, 5],
+                        'y': [1, 3, 5, 9, 13]
+                    }],
+                    'layout': {
+                        'title': 'Graph of \(y = \frac{x^2 - 2}{4}\)'
```

Comment:
Ah I see, this isn't actually valid in plotly.js. Text needs to be _all_ latex or _none_ latex. 

Let's use this example from the docs: https://plot.ly/python/LaTeX/

```
'title': '$\\sqrt{(n_\\text{c}(t|{T_\\text{early}}))}$'
```